domains/certificates/chart/values.yaml

```diff
@@ -1,6 +1,9 @@
 api:
   replicaCount: 1
   image: {}
+registryconnector:
+  replicaCount: 3
```

Comment:
is 3 the "standard"?

Reply:
It seems to be the number used in other domains (e.g. Emissions and Measurements). I just want a number larger than 1.